deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -477,7 +476,12 @@ async def create_new_job(
                     upload_file_to_s3(bucket=PULSE3D_UPLOADS_BUCKET, key=key, file=pv_parquet_path)
 
         return JobResponse(
-            id=job_id, user_id=user_id, upload_id=details.upload_id, status="pending", priority=priority
+            id=job_id,
```

Comment:
I also want to just double check about if you still need 443-445 (similar check also being performed in POST/uploads).. this was being used to display to users if limits had been reached or not, if you have another system set up, you might not need to return that error response? 

Following that, is it still set up that the 're-analysis existing recording' is disabled depending on if the credits are used up? We'll also want to disabled the interactive analysis option in the dropdown once it's reached as well.

Reply:
The interactive analysis, new analysis and re-analysis all get disabled when usage limit gets met. I think it would be a good idea to leave the check. If the users find a way around the frontend and kick of a job manually, it will get prevented by this check. Would it be better to keep this for security ?